pkg/skoop/plugin/calico.go

```diff
@@ -37,6 +38,8 @@ type CalicoConfig struct {
 func (c *CalicoConfig) BindFlags(fs *pflag.FlagSet) {
 	fs.StringVarP(&c.Interface, "calico-host-interface", "", "",
 		"Host interface for calico plugin.")
+	fs.IntVarP(&c.NodeMTU, "calico-node-mtu", "", 1500,
```

Comment:
May use `calico-host-mtu` or `calico-host-interface-mtu` to be consistent with `calico-host-interface`